src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle() {
+        super();
+    }
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public int getArea() {
+        return (int) (Math.PI * (radius * radius));
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Figure: circle, area: ");
+        builder.append(getArea());
+        builder.append(" sq.units, radius: ");
+        builder.append(radius);
+        builder.append(" units, color: ");
+        builder.append(super.getColor().toLowerCase());
```

Comment:
If you override getColor() in Circle class, here you are going call to the parent method. This kills polymorphism.
```suggestion
        builder.append(getColor().toLowerCase());
```